compiler-cli/src/new/tests.rs

```diff
@@ -81,7 +92,18 @@ fn new_with_javascript_template() {
     .unwrap();
     creator.run().unwrap();
 
-    insta::glob!(path, "my_project/*.*", |file_path| {
+    insta::glob!(&path, "my_project/*.*", |file_path| {
+        if !file_path.is_dir() {
+            insta::assert_snapshot!(
+                crate::fs::read(
+                    Utf8PathBuf::from_path_buf(file_path.to_path_buf()).expect("Non Utf8 Path"),
+                )
+                .unwrap()
+            );
+        }
+    });
+
+    insta::glob!(&path, "my_project/**/*.gleam", |file_path| {
```

Comment:
Sorry, what has this file been changed for?

Reply:
The existing snapshot tests only test the files at the top-level of the generated project. That means neither `src/<my_project>.gleam` nor `test/<my_project>.gleam` had a test around their content. Given that I was updating one of those files, I figured having a snapshot test would be valuable. Let me know if you'd prefer I remove this change.

Reply:
Done. Note that I'm excluding hidden directories so that we don't look at contents within `.git`.